key-share/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.2.0
+* Add support of HD wallets compatible with BIP-32 and SLIP-10 [#68],
+  [#74], [#75]
+* Prohibit key shares with zero secret share or secret key [#82]
```

Comment:
Should we remove the same log messages from cggmp21/changelog ?

Reply:
It touched cggmp21 in that we had to alter it a bit to support it, so I think it should be in both

Reply:
Well, in the future, I don't think we'll be duplicating changelog messages between the logs